CHANGELOG/CHANGELOG-0.12.md

```diff
@@ -15,7 +15,7 @@ Changes since `v0.12.0`:
 ## v0.12.0
 
 > [!NOTE]
-> This release is not deployable on GKE. GKE users should use Kueue 0.11.4 until 0.12.1 comes out.
+> This release is not deployable on GKE. GKE users should use Kueue 0.11.5 until 0.12.1 comes out.
```

Comment:
```suggestion
> This release is not deployable on GKE. GKE users should use Kueue 0.12.1 or newer.
```
I guess the message is addressed to users of 0.12.0 so we can skip already 0.11.5

Reply:
That makes sense.
Let's just replace it with 0.12.1.